### ISSUE TITLE
Default to 'none'

### ISSUE THREAD

**Issue by mshick** [intuit/auto#1469]
**Is your feature request related to a problem? Please describe.**

I was running `auto version` on a clean, up-to-date pull and getting `patch` back as my semver string. I suspect this is because `patch` is hardcoded as the default. Ideally this would be a no-op, or configurable as a no-op (`releaseType: none`).

**Describe the solution you'd like**

I would like to have the default `auto version` to be configurable. 

**Describe alternatives you've considered**

My autorc has the following, but it doesn't seem to have any effect.

```json
    {
      "name": "type: documentation ✏️",
      "releaseType": "none",
      "changelogTitle": "Documentation",
      "overwrite": true,
      "default": true
    }
```

I chose the documentation arbitrarily as one I wanted to be `none`. Which is to say, the following triggers an error:

```json
    {
      "releaseType": "none",
      "default": true
    }
```

**Additional context**

Commands I ran:

```console
mshick at michaels-mbp in ~/Code/takeshape/auto-demo (trunk)
$ git pull  
From https://github.com/takeshape/auto-demo
 * branch            trunk      -> FETCH_HEAD
Already up to date.

mshick at michaels-mbp in ~/Code/takeshape/auto-demo (trunk)
$ auto version    
patch
```

**Comment by mshick** [comment]
Sorry, I think I had a change in there I hadn't noticed. Things seem to be working as I'd expect now. Thanks!

**Comment by hipstersmoothie** [comment]
I was actually able to reproduce!

**Comment by hipstersmoothie** [comment]
@mshick What you really want though is the [this option](https://intuit.github.io/auto/docs/configuration/autorc#only-publish-with-release-label) which will only publish a release with the `release` label. 

But thanks for finding this bug!

**Comment by mshick** [comment]
haha, yeah, I'm seeing sort of strange results... it seems like a completely clean run will trigger the `patch` while if I happen to have a `skip`ed PR in the hopper then I get the output I'd expect.

**Comment by hipstersmoothie** [comment]
#1470 should fix this!

**Comment by mshick** [comment]
I'm trying to avoid the `release` label approach in my workflow. Would like my release logic to be untethered to any specific PR. But thanks for the suggestion. And quick debugging / fixing!

**Comment by adierkens** [comment]
:rocket: Issue was released in [`v9.50.7`](https://github.com/intuit/auto/releases/tag/v9.50.7) :rocket:
